<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LandDialog</class>
 <widget class="QDialog" name="LandDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="id_land">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>登陆</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_cancel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="id_ip">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>IP：</string>
   </property>
  </widget>
  <widget class="QLabel" name="id_port">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Port：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_ip_edit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_port_edit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="id_text_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>311</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
